gulpfile.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
   buildLangfiles: buildTasks.langfiles,
   buildCompiled: buildTasks.compiled,
   buildAdvancedCompilationTest: buildTasks.advancedCompilationTest,
-  buildTs: buildTasks.buildTypescript,
+  buildJavaScript: buildTasks.javaScript,
+  buildJavaScriptAndDeps: gulp.series(buildTasks.javaScript, buildTasks.deps),
```

Comment:
Is there any reason to build javascript without building deps first? Do we need to export all three of these thigns, or can it be simplified?

Reply:
As noted: `tsc` is relatively fast compared to `closure-make-deps`, and needs to be run after every `.ts` file change, while the latter is only needed after changes to imports.